microsetta_interface/static/css/4_column_flex.css

```diff
@@ -0,0 +1,29 @@
+  * {
```

Comment:
It looks like the contents of this file are copied from somewhere else on the web. Please make sure to add a comment for attribution, and if there is a license to verify it is compatible with BSD-3.